src/include/daos_srv/pool.h

```diff
@@ -190,6 +190,8 @@ int ds_pool_tgt_map_update(struct ds_pool *pool, struct pool_buf *buf,
 			   unsigned int map_version);
 
 int ds_pool_start(uuid_t uuid);
+int
```

Comment:
It's a tough one, and git-clang-format has a bug here where because the whole block is "wrong" initially it gets confused by the indentation and recommends the wrong thing.  IMO best long-term would be to re-format the whole block and then it would become a non-issue, but that's out of scope for development PRs, the other two options are to do as done here or like you say keep the old style.

This is actually the biggest issue we have with clang-format overall, other than lack of uptake.

Reply:
@tanabarr I initially preserved as much as possible the original formatting.
After, some discussion with @ashleypittman , it appears that if we want to have some consistent C coding syntax (as we have in go) we should respect as much as possible the coding guidelines of clang.
However, I have no issue to revert this modification if the coding guidelines is to respect as much as possible the original formatting.